packages/core/src/CoreUtils.ts

```diff
@@ -24,10 +24,16 @@ type JsonMetadataMetaplexFile = Omit<JsonMetadata, "image"> & {
 export const mintNft = async (
   metaplex: Metaplex,
   json: JsonMetadataMetaplexFile,
-  createNftInput: Omit<CreateNftInput, "uri" | "name" | "sellerFeeBasisPoints">
+  createNftInput: Omit<CreateNftInput, "uri" | "name" | "sellerFeeBasisPoints">,
+  priorityFeeLamports: number = 500000,
 ): Promise<TransactionBuilder> => {
   console.info({ json });
   const { uri } = await metaplex.nfts().uploadMetadata(json);
+  const computeBudget = 250000
```

Comment:
Similar comment re: factoring out this constant.

Reply:
This one is dynamic and @Funkatronics was going to add simulation to get this value.